app/helpers/additional_details/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module AdditionalDetails::ApplicationHelper
+
+  def convert_time(time)
+    modifier = ''
+    ((time.first == '0') || (time[0..1] == '10') || (time[0..1] == '11')) ? modifier = ' AM' : modifier = ' PM'
+
+    DateTime.strptime(time.gsub(':', ''), '%H%M').strftime('%l:%M') + modifier
```

Comment:
It's converting military time to regular time. So if a time string has either '0', '10', or '11' as the leading character's then it is in the morning. All else are PM.